docs/UserGuide.md

```diff
@@ -138,25 +138,26 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Deleting an event : `edelete`
 
-Deletes the specified person from the address book.
+Deletes the event from the event list.
 
-Format: `delete INDEX`
+Format: `edelete eid/EVENT_ID`
```

Comment:
hmm again we should decide on whether to use this or `EVENT_ID`. more in favour towards `EVENT_ID` itself since rx already has that in her command summary

Reply:
aight changed